src/main/scala/ru/d4team/chat/services/PersonService.scala

```diff
@@ -0,0 +1,26 @@
+package ru.d4team.chat.services
+
+import ru.d4team.chat.dao.PersonDAO
+import ru.d4team.chat.models.person.Person
+import zio._
+
+import java.util.UUID
+
+object PersonService {
+  val live: URLayer[PersonDAO, PersonService] = ZLayer.fromFunction(new PersonServiceImpl(_))
+}
+
+trait PersonService {
+  def addPerson(person: Person): Task[Person]
+  def findPerson(id: UUID): Task[Option[Person]]
+  def getAll: Task[List[Person]]
+}
+
+final class PersonServiceImpl(dao: PersonDAO) extends PersonService {
```

Comment:
Consider using a case class

For more information, see https://zio.dev/reference/service-pattern/#2-service-implementation

Reply:
Don't go away, I'll fix it right now :)